vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -132,6 +132,7 @@ $global:imagesToPull += @(
     "mcr.microsoft.com/containernetworking/cni-dropgz:v0.1.4"
     "mcr.microsoft.com/containernetworking/cni-dropgz:v0.2.0"
     "mcr.microsoft.com/containernetworking/azure-cni:v1.5.26"
```

Comment:
I am assuming we need to keep 1.5.26 until rollout of 1.5.32 is complete? Please remember to remove 1.5.26 when it becomes possible :)